### DISCUSSION TITLE
Custom Recipe function.

### DISCUSSION THREAD

**Discussion by BornToSleep7** [Zrips/CMI#7304] [Q&A]
Hello there.
I had been using custom recipe command for quite a time and I would like to know what  exactly does the check box entail/change.

![image](https://user-images.githubusercontent.com/115896586/196003841-5ac6dd44-f1dc-4083-97c8-05a94caf2066.png)

Whenever the it is set to true or false I couldn't see the difference. The question is what ingredients are seen as not exact? Does it mean that there are ingredients that fall into specific groups like planks or stone tiered blocks in vanila crafting? What groups are available?
We are between making horrendous amounts  recipes for items so similar materials can be used or finding a shortcut so nobody has endure click fest that will be brought by doing 9 times more work just because there are nine log and plank types.

**Comment by CloudeLecaw** [comment]
The answer might be a bit more simple than you thought ;) Its a setting you might need if your using custom materials.

If your usung a recipe that needs 5 coals you called &6Happy Coal and this setting is true. Then the Recipe only work with 5 coals called &6Happy Coal.
If the setting is false the player can use each 5 coals he has somewhere.. even the vanilla coal (not wooden coal only the normal one) to craft the recipe.

**Comment by TomLewis** [comment]
Sadly there is a bug, where it will not save a custom item as the output item.

https://github.com/Zrips/CMI/issues/7816
